src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -521,9 +572,9 @@ redeemGas env cmd initialGas cmdResult gid prevLogs mcache = do
     let totalGas   = initialGas + _crGas cmdResult
         fee        = gasFeeOf totalGas (gasPriceOf cmd)
         rk         = cmdToRequestKey cmd
-        initState  = set (evalRefs . rsLoadedModules) mcache
+        initState  = initStateInterpreter $ setModuleCache mcache
           $ initCapabilities [magic_FUND_TX]
-
+    -- print ("\n\nREDEEM GAS:",rk,initialGas,_crGas cmdResult,fee, "\n\n")
```

Comment:
there's a few print statements here we can remove unless you need them for debugging a next PR

Reply:
Yeah they're helpful for gas debugging while we're digging